### ISSUE TITLE
Add better tracking of version compatibility to each app

### ISSUE THREAD

**Issue by mandli** [clawpack/apps#55]
At the very least keep a README in each app that mentions compatibility and gets updated when a new version is released (see #54).

**Comment by rjleveque** [comment]
Good idea - I've started adding a Version section to notebooks, e.g. [chile2010a](http://www.clawpack.org/gallery/_static/apps/notebooks/geoclaw/chile2010a/chile2010a.html), and to the `README.rst` files of Fortran apps.  But not all have this yet.
